vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+---
+title: "Using `epidist` to estimate stratified delays between symptom onset and positive test for the 2014-2016 Ebola outbreak in Sierra Leone"
+description: "A more detailed guide to using the `epidist` R package"
+output: 
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+    number_sections: true
+pkgdown:
+  as_is: true
+# csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Getting in depth with epidist}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+---
+
+```{r setup, include=FALSE}
+# exclude compile warnings from cmdstanr
+knitr::opts_chunk$set(
+  fig.path = "figures/epidist-",
+  cache = TRUE,
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE,
+  warning = FALSE,
+  error = FALSE
+)
+```
+
+The 2014-2016 outbreak of Ebola in West Africa began in Guinea, and quickly spread to neighbouring Liberia and Sierra Leone.
+It was the largest outbreak in the disease's history, and resulted in over 28,600 infections and 11,325 deaths [@who_ebola_2014_2016].
+
+In this vignette, we use the `epidist` package to analyze line list data from the outbreak in Sierra Leone.
+This data was collated by @fang2016transmission.
+We provide the data in the `epidist` package via `?sierra_leone_ebola_data`.
```

Comment:
if the data is just used for examples CRAN might get funny about us including it like this. This is more a note for future pain than any action

Reply:
How can they tell if we just use it for examples or not?

Reply:
they read through the code

Reply:
https://github.com/epinowcast/epidist/issues/279